general/clients/codecs.md

```diff
@@ -0,0 +1,70 @@
+ 
+## [Video Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats "Wikipedia's video codec tables")
+
+||WebOS|Android|AndroidTV|Kodi|[Roku](https://developer.roku.com/docs/specs/streaming.md)|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|[H.262/Xvid](https://en.wikipedia.org/wiki/MPEG-4_Part_2)|✅|✅|✅|✅|✅|
+|[H.262/DivX](https://en.wikipedia.org/wiki/DivX)|✅|✅|✅|✅|✅|
+|[H.264/AVC](https://caniuse.com/#feat=mpeg4 "H264 Browser Support Reference")|🔶<sup>3</sup>|✅|✅|✅|✅|
+|[H.265/HEVC](https://caniuse.com/#feat=hevc "HEVC Browser Support Reference")|❌<sup>1</sup>|🔶<sup>2</sup>|❌|✅|❌|
+
+<sup>1</sup>HEVC support is potentially possible by offloading to the OS. *untested*
+
+<sup>2</sup>Android playback is currently broken. Client reports that HEVC is supported and attempts to Directstream it.
+
+<sup>3</sup>H.264 10Bit is unsupported.
+
+## [Audio Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats#Audio_coding_formats_support "Wikipedia's audio codec tables")
+
+If the audio codec is unsupported or incompatible (such as playing a 5.1 channel stream on a stereo device), the audio codec must be transcoded. This is not nearly as intensive as video coding.
+
+||WebOS|Android|AndroidTV|Kodi|Roku
+|:---:|:---:|:---:|:---:|:---:|:---:
+|MP3|🔶<sup>1</sup>|✅|✅|✅|✅
+|AAC|✅|✅|✅|✅|✅
+|AC3|✅|✅|✅|✅|✅
+|EAC3<sup>2</sup>|✅|✅|✅|✅|✅
+|VORBIS|❌|✅|✅|✅|✅
+
+
+<sup>1</sup>MP3 Mono is incorrectly reported as unsupported.
+
+<sup>2</sup>Only EAC3 2.0 has been tested.
+
+## [Subtitle Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats#Subtitle/caption_formats_support "Wikipedia's subtitle codec tables")
+
+Subtiles can be a subtle issue for transcoding. Containers have a limited number of subtitles that are supported. If subtitles need to be transcoded, it will happen one of two ways. They can be converted into another supported format (text-based subtitles) or burned into the video (image/lossless based and ASS based) due to the subtitles transcoding not being supported. This is the most intenstive method of transcoding due to two transcodings happening at once; applying the subtitle layer on top of the video layer. 
+
+||Format|TS|MP4|MKV|AVI|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|SubRip Text (SRT)|Formatted Text|❌|🔶|✅|🔶|
+|ASS/SSA|Formatted Text|❌|❌|✅|🔶|
+|VobSub|Picture|❌|✅|✅|🔶|
+|DVB-SUB|Picture|✅|❌|✅|❌|
+|MP4TT/TXTT|XML|❌|✅|❌|❌|
+
+Note: ASS Subtitles are only supported by mkv files. Mkv files can't natively be streamed therefore ASS subtitles will always inherently be burned into the video. This is not a limitation of JF. 
```

Comment:
maybe use the `<sus>` thing here like you did above, or at least the formatted note element

Reply:
i just added the subtitles table, i haven't had a chance yet. there's so many rules with subtitles, a third of my notes are on subs alone lol